### PR TITLE
Add CLA and signoff check

### PR DESCRIPTION
**Pull request by viadea**
Add CLA and signoff check

### CODE REVIEW

File: .github/workflow/signoff-check.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
Comment (firestarman) [inline .github/workflow/signoff-check.yml]: The year should be '2021'.

File: .github/workflow/sigoff-check/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
Comment (firestarman) [inline .github/workflow/sigoff-check/Dockerfile]: The year should be '2021'.

File: .github/workflow/sigoff-check/action.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
Comment (firestarman) [inline .github/workflow/sigoff-check/action.yml]: The year should be '2021'.

File: .github/workflow/sigoff-check/signoff-check [hunk 4]
```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020, NVIDIA CORPORATION.
```
Comment (firestarman) [inline .github/workflow/sigoff-check/signoff-check]: The year should be '2021'.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+# Contributing to Spark Examples
+
+### Sign your work
+
+We require that all contributors sign-off on their commits. This certifies that the contribution is your original work, or you have rights to submit it under the same license, or a compatible license.
```
Comment (sameerz) [inline CONTRIBUTING.md]: I suggest that lines be 100 characters or less for ease of readability

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+(c) The contribution was provided directly to me by some other
+    person who certified (a), (b) or (c) and I have not modified
+    it.
+
+(d) I understand and agree that this project and the contribution
+    are public and that a record of the contribution (including all
+    personal information I submit with it, including my sign-off) is
+    maintained indefinitely and may be redistributed consistent with
+    this project or the open source license(s) involved.
+```
```
Comment (sameerz) [inline CONTRIBUTING.md:72]: Add a note to describe where this signoff file was derived from

Comment (viadea) [reply CONTRIBUTING.md:72]: @sameerz  Just added a link pointing to https://github.com/NVIDIA/spark-rapids . How do you think now?

File: .github/workflow/sigoff-check/action.yml (hunk 3 above)
Comment (viadea) [reply .github/workflow/sigoff-check/action.yml]: @firestarman I just changed all 2020 to 2021. How do you think now? thx

File: CONTRIBUTING.md (hunk 6 above)
Comment (sameerz) [reply CONTRIBUTING.md:72]: Looks good

**Review by sameerz — APPROVED** [review]

**Comment by firestarman** [comment]
LGTM

**Review by pxLi — APPROVED** [review]
LGTM

**Pull request merged**